src/iam/iam-base.ts

```diff
@@ -298,7 +299,10 @@ export class IAMBase {
      * @returns boolean that indicates the session state
      */
     public isSessionActive() {
-        return Boolean(this._publicKey) && Boolean(this._providerType);
+        return (
+            (Boolean(this._publicKey) && Boolean(this._providerType)) ||
```

Comment:
Is this change because of missing provider of type Wallet? If so then it would be more consistent to add this type

Reply:
There is no wallet type which means there is no providerType. What should i put there? Because localStorage is used only by browser and i can't use only publicKey.

Reply:
You can add Wallet providerType. And to avoid storing session in Node.js environment you can check environment before `storeSession`